src/TAlias.cpp

```diff
@@ -110,7 +110,7 @@ bool TAlias::match(const QString& toMatch)
 
     //cout <<" LINE="<<subject<<endl;
     if (mRegexCode.size() > 0) {
-        rc = pcre_exec(re.data(), 0, subject, subject_length, 0, 0, ovector, 100);
+        rc = pcre_exec(re.data(), nullptr, subject, subject_length, 0, 0, ovector, 100);
```

Comment:
`nullptr` is fine in the context of being passed to `C` functions. Using `NULL` provides no benefits.